Detectors/PHOS/reconstruction/src/RawReaderMemory.cxx

```diff
@@ -40,6 +40,8 @@ o2::header::RDHAny RawReaderMemory::decodeRawHeader(const void* payloadwords)
     return o2::header::RDHAny(*reinterpret_cast<const o2::header::RAWDataHeaderV5*>(payloadwords));
   } else if (headerversion == 6) {
     return o2::header::RDHAny(*reinterpret_cast<const o2::header::RAWDataHeaderV6*>(payloadwords));
+  } else if (headerversion == 7) {
+    return o2::header::RDHAny(*reinterpret_cast<const o2::header::RAWDataHeaderV7*>(payloadwords));
   }
```

Comment:
This will work but in principle, you don't need all this `if`'s (in the function which is called very often) and header copy, you can do simply 
```
const o2::header::RDHAny* RawReaderMemory::decodeRawHeader(const void* payloadwords) const
  auto rdh =  reinterpret_cast<const o2::header::RDHAny*>(&payloadwords);
  // in case you want to check the header version:
  auto v = RDHDecoder::getVersion(*rdh);
  if (v < RDHDecoder::getVersion<o2::header::RDHLowest>() || v>RDHDecoder::getVersion<o2::header::RDHHighest>()) {
     LOG(error) << "Wrong header version " << v;
     throw RawDecodingError::ErrorType_t::HEADER_DECODING;
  }
  return rdh;
```